Authors.md

```diff
@@ -1,4 +1,3 @@
 # Authors
 
-- Spencer Kaiser (spencer.kaiser@aa.com)
-- John Kahn (john.kahn@aa.com)
\ No newline at end of file
+- Henry Nguyen (henry.nguyen@aa.com)
```

Comment:
@henryn098 in the future, don't remove others from this file 😉 I just realized we messed up our Authors file with this commit